packages/gatsby-plugin-gatsby-cloud/src/build-headers-program.js

```diff
@@ -313,6 +314,15 @@ const writeHeadersFile =
   ({ publicFolder }) =>
   contents =>
     new Promise((resolve, reject) => {
+      /**
+       * Emit Headers via IPC
+       */
+      Object.entries(contents).map(([k, val]) => {
+        emitHeaders({
+          [k]: val,
+        })
```

Comment:
This is a personal comment so do with it what you want. IPC apis are similar to redux actions and being explicit is mostly better as it allows us to add extra args later on.
```suggestion
        emitHeaders({
          url: k,
          headers: val
        })
```